main.go

```diff
@@ -222,6 +222,11 @@ func setupGlobals() {
 	if config.Global().NewRelic.AppName != "" {
 		NewRelicApplication = SetupNewRelic()
 	}
+
+	if globalConfig := config.Global(); globalConfig.PaginationItemsPerPage <= 0 {
+		globalConfig.PaginationItemsPerPage = 10
```

Comment:
I think it is better move default value `10` to constant, to eliminate "magic" numbers in code base